packages/@uppy/aws-s3-multipart/src/MultipartUploader.js

```diff
@@ -162,6 +171,9 @@ class MultipartUploader {
       if (!this.#abortController.signal.aborted) this.#abortController.abort(pausingUploadReason)
       this.#abortController = new AbortController()
       this.#resumeUpload()
+    } else if (this.#isRestoring) {
```

Comment:
in this `start` function, the difference seems a bit to subtle between "restoring" and "resuming". Perhaps we can name this differences more explicitly if you make it specifically for golden retriever.